packages/transport-webrtc/src/index.ts

```diff
@@ -3,6 +3,40 @@ import { WebRTCDirectTransport, type WebRTCTransportDirectInit, type WebRTCDirec
 import type { WebRTCTransportComponents, WebRTCTransportInit } from './private-to-private/transport.js'
 import type { Transport } from '@libp2p/interface/transport'
 
+export interface DataChannelOptions {
+  /**
+   * The maximum message size sendable over the channel
+   */
+  maxMessageSize?: number
+
+  /**
+   * If the channel's `bufferedAmount` grows over this amount in bytes, wait
```

Comment:
For these options, it would be good to make sure the unit of each item is explicitly commented. Eg: bytes, milliseconds, etc

Reply:
Done!

Reply:
Part of me wishes we had the convention of suffixing options with units, e.g. `timeoutMs` but here we are.